apps/expo-go/android/tools/build.gradle

```diff
@@ -2,6 +2,9 @@ apply plugin: 'java'
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
+  implementation "org.jetbrains.kotlin:kotlin-compiler-embeddable:1.8.0"
+  implementation "org.jetbrains.kotlin:kotlin-compiler:1.8.0"
+  implementation "org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.8.0"
```

Comment:
quick question: do we need these for ReactAndroidCodeTransformer? wondering if that would bring to runtime cost?

Reply:
Actually, this won't be necessary for now, taking a deeper look at the Kotlin files I notice that we won't need the visitor function for Kotlin files for now, we can keep with just using the modifySource